conf/cassandra.yaml

```diff
@@ -2113,27 +2113,27 @@ drop_compact_storage_enabled: false
+# For example, if we're going to remain compatible with Cassandra 4.x, the value of this property should be 4, which
+# will make us use sstables in the latest N version of the BIG format.
 #
-# This will also determine if certain features depending on newer formats are available. For example, extended TTLs
-# up to 2106 depend on the sstable, commitlog, hints and messaging versions that were introduced by Cassandra 5.0,
-# so that feature won't be available if this property is set to CASSANDRA_4. See upgrade guides for details. Currently 
-# the only supported major is CASSANDRA_4.
+# This will also determine if certain features that depend on newer formats are available. For example, extended TTL
+# (up to 2106) depends on the sstable, commit-log, hints, and messaging versions introduced by Cassandra 5.0, so that
+# feature won't be available if this property is set to CASSANDRA_4. See the upgrade guide for more details.
 #
-# Possible values are in the StorageCompatibilityMode.java file accessible online. At the time of writing these are:
+# Possible values are:
 # - CASSANDRA_4: Stays compatible with the 4.x line in features, formats and component versions.
-# - UPGRADING:   The cluster monitors nodes versions during this interim stage. _This has a cost_ but ensures any new features, 
-#                formats, versions, etc are enabled safely.
+# - UPGRADING:   The cluster monitors the version of each node during this interim stage. This has a cost but ensures
+#                all new features, formats, versions, etc., are enabled safely.
 # - NONE:        Start with all the new features and formats enabled.
 #
 # A typical upgrade would be:
-# - Do a rolling upgrade starting all nodes in CASSANDRA_Y compatibility mode.
-# - Once the new binary is rendered stable do a rolling restart with UPGRADING. The cluster will enable new features in a safe way 
-#   until all nodes are started in UPGRADING, then all new features are enabled.
-# - Do a rolling restart with all nodes starting with NONE. This sheds the extra cost of checking nodes versions and ensures 
-#   a stable cluster. If a node from a previous version was started by accident we won't any longer toggle behaviors as when UPGRADING.
+# - Do a rolling upgrade, starting all nodes in CASSANDRA_4 compatibility mode.
+# - Once the new binary is rendered stable, do a rolling restart with the UPGRADING mode. The cluster will keep new
+#   features disabled until all nodes are started in the UPGRADING mode; when that happens, new features controlled by
+#   the storage compatibility mode are enabled.
+# - Do a rolling restart with all nodes starting with the NONE mode. This eliminates the cost of checking node versions
+#   and ensures stability. Such a node will not disable any features even when a node running Cassandra 4 tries to
+#   contact it (as it would do in the UPGRADING mode).
```

Comment:
I think the previous sentence `If a node from a previous version was started by accident we won't any longer toggle behaviors as when UPGRADING` reads clearer and I wouldn't mention a specific version for the same reasons as before